lib/transformers/transformers.js

```diff
@@ -8,7 +8,15 @@ const url = require('url');
 const isHealth = require('./isHealth');
 const { extractHeaders, truncateText } = require('../utils');
 
-const mapValuesDeep = (v, callback) => (isObject(v) ? mapValues(v, v => mapValuesDeep(v, callback)) : callback(v));
+const mapValuesDeep = (v, callback) => {
+  if (v && Array.isArray(v)) {
+    return v.map(vv => mapValuesDeep(vv, callback));
+  } else if (isObject(v)) {
+    return mapValues(v, vv => mapValuesDeep(vv, callback));
+  } else {
+    return callback(v);
+  }
```

Comment:
❓ This is noop right? It worked this way before correct?

Reply:
No it wasn't. Well, It depends 😛 

It was converting arrays to 
```
{
    '0': {...},
    '1': {...},
    '2': {...}
}
```